packages/devtools_server/lib/src/handlers.dart

```diff
@@ -103,3 +177,182 @@ class ServerApi {
   FutureOr<shelf.Response> notImplemented(shelf.Request request) =>
       shelf.Response(204);
 }
+
+// Access the ~/.flutter file.
+class Usage {
+  Analytics _analytics;
+
+  /// Create a new Usage instance; [versionOverride] and [configDirOverride] are
+  /// used for testing.
+  Usage(
+      {String settingsName = 'flutter',
+      String versionOverride,
+      String configDirOverride}) {
+    // final FlutterVersion flutterVersion = FlutterVersion.instance;
+    // final String version = versionOverride ?? flutterVersion.getVersionString(redactUnknownBranches: true);
```

Comment:
is this commented out code necessary?

Reply:
Removed was my notes on things to do.  No longer needed. gone.